src/Raven.Client/Documents/Operations/TimeSeries/GetTimeSeriesOperation.cs

```diff
@@ -62,12 +59,27 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                     .Append("id=")
                     .Append(Uri.EscapeDataString(_docId))
                     .Append("&name=")
-                    .Append(Uri.EscapeDataString(_timeseries))
-                    .Append("&from=")
-                    .Append(_from.ToString("o"))
-                    .Append("&to=")
-                    .Append(_to.ToString("o"));
-                
+                    .Append(Uri.EscapeDataString(_timeseries));
+
+                if (_ranges != null)
+                {
+                    foreach (var range in _ranges)
+                    {
+                        var f = range.From.Kind == DateTimeKind.Local
+                            ? DateTime.SpecifyKind(range.From, DateTimeKind.Unspecified)
+                            : range.From;
+
+                        var t = range.To.Kind == DateTimeKind.Local
+                            ? DateTime.SpecifyKind(range.To, DateTimeKind.Unspecified)
+                            : range.To;
+
+                        pathBuilder.Append("&from=")
+                            .Append(f.ToString("o"))
```

Comment:
Shouldn't we use `DefaultFormat.DateTimeFormatsToWrite` ?

Reply:
Does Kestrel have max request query string?

Reply:
Not small enough that we would care about, I think.
The usual numbers are around 1 - 2 KB

Reply:
We set it to 16kb, you can change that via configuration: `Http.MaxRequestLineSizeInKb` so it should not be a problem.

When it comes to format I see that in other places we are using e.g.
```csharp
pathBuilder.Append("&before=").Append(_before.Value.GetDefaultRavenFormat(_before.Value.Kind == DateTimeKind.Utc));
```

Reply:
done